src/main/java/com/dreammaster/main/XMod.java

```diff
@@ -0,0 +1,8 @@
+package com.dreammaster.main;
+
+import cpw.mods.fml.common.Loader;
+
+public class XMod {
+
+    public static final boolean DETRAVSCANNERMOD = Loader.isModLoaded("detravscannermod");
```

Comment:
Isn't detrav in the GT enum?

Reply:
nope

Reply:
I can make a PR there if you want though

Reply:
[PR out](https://github.com/GTNewHorizons/GT5-Unofficial/pull/2663)

Reply:
merged